### PR TITLE
Add link to OTP developer group.

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
This PR adds a link to the application footer to the OpenTripPlanner developer Google Group.

A new configuration parameter ~`OTP_DEVGROUP_URL`~ `OTP_FORUM_URL` contains the link to the OTP developers ~group~ forum. (Once deployed, remember to add this parameter to Vercel).

### CODE REVIEW

File: .env.tmp [hunk 1]
```diff
@@ -5,7 +5,9 @@ API_KEY='your-api-key'
 AUTH0_AUDIENCE='https://otp-middleware'
 AUTH0_DOMAIN='you.auth0.com'
 AUTH0_CLIENT_ID='your-client-id'
+OTP_DEVGROUP_URL='https://groups.google.com/g/opentripplanner-dev'
 OTP_UI_URL='https://otp-ui'
+SITE_LOGO='https://path/to/logo.png'
```
Comment (evansiroky) [inline .env.tmp:10]: What does this do? Is there a way to add comments to this file?

Comment (evansiroky) [reply .env.tmp:10]: Ping @binh-dam-ibigroup

Comment (binh-dam-ibigroup) [reply .env.tmp:10]: Hmm for some reason I thought I answered already. `SITE_LOGO` was introduced by https://github.com/ibi-group/otp-admin-ui/commit/24045651bcde1c8b7399430dfbe1b6fe31441a17, and reviewing variables in the config template file made me add it as it was missing.

File: components/Footer.js [hunk 2]
```diff
@@ -55,6 +55,14 @@ export default function Footer (props) {
                 API documentation
               </NavLink>
             </li>
+            <li>
+              <NavLink
+                href={process.env.OTP_DEVGROUP_URL}
+                external
+              >
+                OpenTripPlanner Developers Google Group
```
Comment (landonreed) [inline components/Footer.js:63]: I think we should call this "OTP Developers Forum". It fits the container a little better and the project requirement requires us to provide a "developers forum", so I think it matches that requirement a little better also.

**Review by landonreed — APPROVED** [review]
Looks good, conditional approval on link text tweak.

File: components/Footer.js (hunk 2 above)
Comment (binh-dam-ibigroup) [reply components/Footer.js]: Updated in https://github.com/ibi-group/otp-admin-ui/pull/28/commits/465cffdbf274a02b25f2c7500a568c2daf349046.

**Review by evansiroky — APPROVED** [review]

**Pull request merged**